README.md

```diff
@@ -82,6 +99,9 @@ body that says "hello".
 Your Answer:
 ```js
 // your code here
+$"(#greeting").click(function(){
+  ("body").append("<p>Hello</p>");
+})
 ```
```

Comment:
`("body")` needs a `$` before it too if it's going to be treated as a jQuery object.